indico/modules/events/management/templates/clone_event.html

```diff
@@ -0,0 +1,107 @@
+{% from 'forms/_form.html' import form_header, form_fieldset, form_footer, form_row, form_rows %}
+{% from 'message_box.html' import message_box %}
+
+{% macro invisible_field(field) -%}
+    {% set field_value = field._value() if field._value else field.data %}
+    {% set data = [field_value] if field_value is string else ([field_value|tojson] if field_value is mapping else field_value) %}
```

Comment:
that line is very readable ;) I'd wrap it and also move the `|forceescape` behind the `|tojson` (since that's the only case where it's needed)